bw2io/backup.py

```diff
@@ -61,25 +71,38 @@ def backup_project_directory(project: str):
     if project not in projects:
         raise ValueError("Project {} does not exist".format(project))
 
+    dir_backup = dir_backup or os.path.expanduser("~")
+
+    if timestamp:
+        timestamp = f'.{datetime.datetime.now().strftime("%d-%B-%Y-%I-%M%p")}'
+    else:
+        timestamp = ""
+
     fp = os.path.join(
-        os.path.expanduser("~"),
-        "brightway2-project-{}-backup.{}.tar.gz".format(
-            project, datetime.datetime.now().strftime("%d-%B-%Y-%I-%M%p")
-        ),
+        dir_backup, f"brightway2-project-{project}-backup{timestamp}.tar.gz"
```

Comment:
This is missing a `.`

Reply:
actually there is a leading dot in the the f-string that is the timestamp variable
if the timestamp is not desired, it is an empty string, so then ...-backup.tar.gz